BestPractices.md

```diff
@@ -177,3 +185,35 @@ internal final class MyClass {
 Retain cycle doesn't apply to Swift structs since they are passed by value and not by reference.
 
 *Rationale* Retain cycle bugs are very easy to reproduce, being very careful when manipulating pointers is crucial to build a solid app.
+
+### Documentation ###
+
+Documentation is part of Prolific Swift Style Guide, here are some best practices that can be applied to improve your documentation.
+
+#### Format ####
+
+Apple provides two ways to document your code. You can use either `///` to have a one line documentation, or `/** */` to have mulitple lines documentation. Most of the time prefer use the `/** */` as it allows more flexibility in your documentation.
```

Comment:
I'd say drop the "prefer /** */" thing because I think that's subjective and it's kind of silly to just say that. Instead, I'd rather we just link here to Apple's root documentation about apple doc because it's more relevant. I'd prefer that instead of pointing out sporadically cool appledoc features like "seealso", etc.